packages/aws-cdk/README.md

```diff
@@ -560,6 +561,111 @@ This feature is currently in preview. Be aware of the following limitations:
   in the right order. The CLI will not help you import dependent resources in the right
   order, the CloudFormation deployment will fail with unresolved references.
 
+
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ 
+
+CDK migrate is currently experimental and may have breaking changes in the future. 
+
+Generates a CDK application from an existing JSON/YAML CloudFormation template.
+Takes a valid CloudFormation template as input from either a local file specified with `--from-path`,
+or from a deployed Cloudformation stack using `--from-stack`. Generates a CDK application 
+which will synthesize a CloudFormation template with identical resource configurations to the provided template. 
+The generated application will be initialized in the current working directory with a single stack where 
+the stack, app, and directory will all be named using the provided `--stack-name`. The generated application will 
+be within a generated subdirectory in your current working directory unless `--output-path` is specified. 
+All CDK supported lanaguages are supported, language choice can be specified with `--language`.
+
+#### Generate a typescript application from a local template.json file
+
+```console
+$ # template.json is a valid cloudformation template in the local directory
+$ cdk migrate --from-path ./template.json --stack-name MyAwesomeApplication
+```
+
+This command will generate a new direcotry named `MyAwesomeApplication` within your current working directory, and 
+then initialize a new CDK application within that directory which has the same resource configuration as the
+as the provided template.json
+
+This results in a CDK application with the following structure, where the lib directory contains a stack definition
+with the same resource configuration as the provided template.json.
+
+```console
+├── README.md
+├── bin
+│   └── my_awesome_application.ts
+├── cdk.json
+├── cdk.out
+│   ├── MyAwesomeApplication.assets.json
+│   ├── MyAwesomeApplication.template.json
+│   ├── cdk.out
+│   ├── manifest.json
+│   ├── synth.lock
+│   └── tree.json
+├── jest.config.js
+├── lib
+│   └── my_awesome_application-stack.ts
+```
+
+#### Generate a python application from a deployed stack
+
+If you already had a cloudformation stack deployed in your account and would like to migrate to using CDK instead of Cloudformation.
```

Comment:
```suggestion
If you already had a CloudFormation stack deployed in your account and would like to migrate to using CDK instead of CloudFormation.
```